api/src/main/java/io/opencensus/trace/propagation/SpanContextParseException.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.trace.propagation;
+
+/** Exception thrown when a {@link io.opencensus.trace.SpanContext} cannot be parsed. */
+public class SpanContextParseException extends Exception {
```

Comment:
Should this class be final?  I made TagContextParseException final because I thought that subclasses would only be useful if we added them to the API, so that users could catch a subtype.  If we decided to add subclasses to the API, we could make TagContextParseException non-final at the same time.

Reply:
Done.